dist/bin/node

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+set -eo pipefail
+
+# Custom script to replace node and run with V8 flags that make the execution of the
+# benchmarks more predictable.
+# Depending on the version of node, some flags may be deprecated.
+
+# Retrieve the original path by removing the folder containing CodSpeedHQ/action from the path.
+ORIGINAL_PATH=$(echo "$PATH" | tr ":" "\n" | grep -v "CodSpeedHQ/action" | tr "\n" ":")
+# Check if node is in the path.
+if ! command -v node &>/dev/null; then
+    echo "Error: node not found in PATH. There might be a problem with the node installation."
+    exit 1
+fi
```

Comment:
This should use `REAL_NODE_PATH` here it will always give true since the script itself is in the path

Reply:
Nicely spotted

```
if ! env PATH="$ORIGINAL_PATH" command -v node &>/dev/null; then
    echo "Error: node not found in PATH. There might be a problem with the node installation."
    exit 1
fi
```

Should do the trick